docusaurus/docs/guides/Identity_Providers/autheila/index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Authelia for BrowZer
+sidebar_label: Authelia
+slug: /identity-providers-for-browZer-authelia
+---
+
+<head>
+  <title>Authelia for OpenZiti BrowZer</title>
+  <meta
+    name="description"
+    content="How to configure Authelia for OpenZiti BrowZer."
+  />
+</head>
+
+<img src="/icons/logo-authelia.svg" alt="How to configure Authelia for OpenZiti BrowZer" width="20%"/>
+
+### Express Your Interest in this Identity Provider
+If you want to use this identity provider with your BrowZer network, please express your interest over in our [Discourse forum](https://openziti.discourse.group/).
```

Comment:
all of these sections might be better served from a docusaurus "partial" where you can include the exact same content in a page as opposed to duplicating it. Please consider extracting this into a partial. An example is `./_linux-package-repo.mdx` and 
```
guides/deployments/10-linux/10-controller/10-deploy.mdx
guides/deployments/10-linux/20-router/10-deploy.mdx
```

Reply:
Done.